pages/infrastructure_providers-validators/running_full_node.md

```diff
@@ -33,28 +36,38 @@ From the list of healthy peers that you retrieve from peer node provider, choose
+### Save an Address Book File
+Save an address book file of trusted nodes operated by a provider to more quickly and reliably connect to the network.
 
-Save the `addrbook.json` file in your `/.dydxprotocol/config` directory.
-
-## Restore Your Node
-> If you followed the procedure on the previous page, [Set Up a Full Node](../infrastructure_providers-validators/how_to_set_up_full_node.md), you already have a snapshot installed.
+Download an up-to-date `addrbook.json` file for your deployment from an [Address Book](../infrastructure_providers-network/resources.mdx#address-book-providers) provider. 
 
-Your full node needs a backup plan to replay the history of the network in case it falls out of sync. You can prepare a backup for your node in one of two ways.
-
-### Snapshot
-You can use a **snapshot** stored on the system that your node runs on. A snapshot contains a compressed copy of the application state at the time the snapshot was taken. If your node falls out of sync, a snapshot allows the node to recover to that saved state before replaying the rest of the history of the network. This speeds up the syncing process because you avoid replaying the entire history of the network, instead starting from your stored application state snapshot. To use a snapshot to back up your full node, install a snapshot for your deployment from a [Snapshot Service](../infrastructure_providers-network/resources.mdx#snapshot-service).
-
-### State Sync
-You can use **state sync**, a set of configuration settings that allow your node to retrieve a snapshot from the network. If your node falls out of sync, it queries a state sync node for a verified, recent snapshot of the application state. This speeds up the syncing process because you avoid replaying the entire history of the network, instead starting from the network's most recent application state snapshot. To use state sync to back up your full node, follow the instructions for your deployment from a [State Sync](../infrastructure_providers-network/resources.mdx#state-sync-service) service.
+Save the `addrbook.json` file in your `/.dydxprotocol/config` directory.
 
-## Optimize Pruning Settings
+### Configure a Pruning Strategy
 In general, dYdX recommends the following pruning setting, configured in your `app.toml` file:
 
 ```bash
 # app.toml
 pruning = "everything" # 2 latest states will be kept; pruning at 10 block intervals
 ```
 
-However, if you want to use your node to query historical data, configure a custom pruning strategy to retain more states. Retaining more states increases storage requirements.
\ No newline at end of file
+However, if you want to use your node to query historical data, configure a custom pruning strategy to retain more states. Retaining more states increases storage requirements.
+
+## Prepare to Restore Your Node
```

Comment:
Reframed this section from "Restore your Node" -- the page is about optimizing your node to keep a healthy connection, so this section is about configuring your system so that you can restore right away/ automatically if you need to. There should be procedures on another page and we should link to them from here when they're done